### PR TITLE
changes horrible blue to lovely teal

### PR DESCRIPTION
**Pull request by mrniaboc**
Fixes # .

Describe your changes.

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch?

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?

### CODE REVIEW

**Comment by mrniaboc** [comment]
Check it out on staging https://header-colour-change.pfe-preview.zooniverse.org/projects

File: app/layout/index.styl [hunk 1]
```diff
@@ -23,7 +23,7 @@
     z-index: 2
 
   &__header
-    background-color: #0072ff
+    background-color: #00979D
```
Comment (rogerhutchings) [inline app/layout/index.styl]: Use `LIGHT_GREEN_HEADER` instead of using the hex code again?

File: css/common.styl [hunk 2]
```diff
@@ -7,14 +7,16 @@ COBOLT_BLUE = #0072ff
...
+LIGHT_GREEN_HEADER = #00979D
 
 LIGHT_GREY = #CBCCCB
 SUCCESS = #009E0C
 WARNING = #FFB300
 FAILURE = #FF5A43
 
 DISCIPLINE_COLOR_ARTS = #AFA48C
-DISCIPLINE_COLOR_ALL = #0072FF
+DISCIPLINE_COLOR_ALL = #00979D
```
Comment (rogerhutchings) [inline css/common.styl]: `DISCIPLINE_COLOR_ALL = LIGHT_GREEN_HEADER`

**Comment by shaunanoordin** [comment]
## Status
Changes LGTM. I have no issues with the design and welcome our new header - teal is the real deal.

Minor comments (non-blocking for this PR):
* Do we want to (eventually) change the hyperlink colours to match the teal theme?
* Do we want to (eventually) change the default project and default user images to match the teal theme?

## Actions
None to be taken; waiting for @grant to respond to @rogerhutchings's comments and for any further comments before merging.

**Comment by jwbmartin** [comment]
@shaunanoordin I def think we should change the hyperlink blue! And I'll see if I can whip up something easy for the default image. Maybe just a gray nondescript human like Facebook and twitter do?

**Comment by shaunanoordin** [comment]
@jwbmartin I'm helping!

zooniverse-avatar-2

zooniverse-avatar-1

zooniverse-avatar-3

**Comment by jwbmartin** [comment]
@shaunanoordin 🙏

**Comment by jwbmartin** [comment]
screen shot 2016-10-10 at 2 52 58 pm
screen shot 2016-10-10 at 2 53 03 pm
screen shot 2016-10-10 at 2 52 53 pm

**Comment by shaunanoordin** [comment]
I vote for the ninja face, no contest.

**Comment by srallen** [comment]
We might want more than one generic placeholder since the face doesn't make sense for projects. 👍 to ninja avatar for users though

**Comment by jwbmartin** [comment]
Right! I forgot about the generic project card

**Comment by jwbmartin** [comment]
Generic things
screen shot 2016-10-10 at 4 10 57 pm
screen shot 2016-10-10 at 4 11 05 pm
screen shot 2016-10-10 at 4 11 00 pm

File: app/layout/index.styl (hunk 1 above)
Comment (shaunanoordin) [inline app/layout/index.styl]: Changing `#00979D` to `LIGHT_GREEN_HEADER` doesn't work, strangely? It seems that `app/layout/index.styl` doesn't automatically get the var values defined in `css/common.styl`

**Comment by shaunanoordin** [comment]
screen shot 2016-10-11 at 17 20 32

The `simple-avatar` looks good, but the `simple-pattern` (used in Collections and Projects) was originally designed to be a [repeating pattern,](https://static.zooniverse.org/www.zooniverse.org/assets/simple-pattern.jpg) so the bottom-level shading and non-aligned edges of the [teal pattern.](https://cloud.githubusercontent.com/assets/14332849/19251279/31171126-8f04-11e6-9af4-a397c3834f14.png)

Of course, one simple solution is to ignore the repeating pattern rule and just use `background-size: cover`, if we don't mind stretching the non-repeating teal pattern to fit the size of the panels.

File: app/layout/index.styl (hunk 1 above)
Comment (rogerhutchings) [inline app/layout/index.styl]: Huh. What's the load order?

**Comment by jwbmartin** [comment]
@shaunanoordin oh yeah these are just screen shots not the actual patterns

**Comment by jwbmartin** [comment]
userprofile_avatar-04
userprofile_avatar-01

**Comment by shaunanoordin** [comment]
Ooh! These look way more awesome. :+1:

**Comment by shaunanoordin** [comment]
## Overview
New Teal header has been complemented with a teal (ninja!) Default Avatar and a teal Default Pattern (for Projects/Collections)

## Preview
screen shot 2016-10-14 at 15 30 02

screen shot 2016-10-14 at 15 30 22

screen shot 2016-10-14 at 15 30 11

## Status
Ready for re-review

**Pull request merged**